CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 3.0)
+
+set(CMAKE_C_COMPILER clang)
+set(CMAKE_CXX_COMPILER clang++)
+
+project(AudacityNoiseReduction VERSION 1.0.0)
+
+add_compile_options(-std=c++11 -Wall -Wextra -pedantic)
```

Comment:
a bit better to use `CXX_STANDARD`
https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html

Reply:
Okay, thanks. I have read about it, but I was not sure if it conflicts with add_compile_options. I assumed it kind of replaces all compiler flags.

Or does it actually add them?

This means if you have something like
```
add_compile_options(-Wall)
add_compile_options(-Wextra)
```

that they are combined?

Reply:
yeah but its not a big deal either way :)